proprietary/tokens/src/components/ams/header.tokens.json

```diff
@@ -1,13 +1,51 @@
 {
   "ams": {
     "header": {
-      "column-gap": {
-        "value": "{ams.space.grid.md}",
-        "comment": "Must have the same value as `ams.grid.column-gap`."
+      "padding-block": { "value": "{ams.space.grid.sm}" },
+      "brand-section": {
+        "column-gap": { "value": "{ams.space.md}" },
+        "row-gap": { "value": "{ams.space.grid.xs}" }
       },
-      "padding-block": { "value": "{ams.space.md}" },
       "logo-link": {
         "outline-offset": { "value": "{ams.focus.outline-offset}" }
+      },
+      "mega-menu": {
+        "button": {
+          "label": {
+            "open": {
+              "font-weight": { "value": "{ams.text.font-weight.bold}" }
+            }
+          }
+        }
+      },
+      "menu": {
+        "column-gap": { "value": "{ams.space.grid.md}" },
+        "row-gap": { "value": "{ams.space.grid.xs}" },
+        "item": {
+          "color": { "value": "{ams.link-appearance.color}" },
+          "column-gap": { "value": "{ams.space.xs}" },
+          "font-family": { "value": "{ams.text.font-family}" },
+          "font-size": { "value": "{ams.text.level.6.font-size}" },
```

Comment:
I've made a remark on level 6 in [the ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-884), it is weird of the Header font size is significantly smaller than the text right below.

Reply:
I agree, let's discuss this with Bas

Reply:
We’ve changed this to use Text Level 5.